integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/ConsumerMain.java

```diff
@@ -116,6 +117,7 @@ private static void testStandalone() throws Throwable {
     ITJUnitUtils.runWithHighwayAndRest(TestChangeTransport.class);
     ITJUnitUtils.runWithHighwayAndRest(TestDataTypePrimitive.class);
     ITJUnitUtils.runWithHighwayAndRest(TestAnnotatedAttribute.class);
+    ITJUnitUtils.runWithHighwayAndRest(TestIgnoreStaticMethod.class);
```

Comment:
too complex, just check meta content is enough, eg: org.apache.servicecomb.it.testcase.TestApiParam

Reply:
ok. i see